client/tink_keyset_helper_test.go

```diff
@@ -99,3 +104,116 @@ func TestConvertTinkKeysetHandleToBytes(t *testing.T) {
 		t.Fatalf("when convert tink keyset handle to bytes, the keyset becomes invalid")
 	}
 }
+
+// getDummyKnoxVersionList is a helper for test. It returns a dummy knox version list for testing and a map from
+// Tink key ID to knox version ID. The data of each version is a Tink keyset in bytes that contains a single Tink
+// key. Argument counts decides how many versions are in this dummy veriosn list. Argument templateFunc decides
+// the type of created Tink keyset in each version.
+func getDummyKnoxVersionList(
+	counts int,
+	templateFunc func() *tinkpb.KeyTemplate,
+) (knox.KeyVersionList, map[uint32]uint64) {
+	var dummyVersionList knox.KeyVersionList
+	tinkKeyIDToKnoxVersionID := make(map[uint32]uint64)
+	// counts decide how many versions this dummy version list will have
+	for i := 0; i < counts; i++ {
+		// get a tink keyset in bytes that contains a fresh single key and the keyID is not duplicated
+		var keysetInbytes []byte
+		for {
+			keysetHandle, err := keyset.NewHandle(templateFunc())
```

Comment:
for this and the other "infinite" loop, it might be cleaner to write it something like:

```
var isDuplicated = true
for isDuplicated {
    generateKey()
    isDuplicated = checkIfDuplicated()
} 
```

Reply:
Thank you for your comment! I revised the code accordingly.